base/dcp_client.go

```diff
@@ -273,14 +273,13 @@ func (dc *DCPClient) Start() (doneChan chan error, err error) {
 	return dc.doneChannel, nil
 }
 
-// Close is used externally to stop the DCP client. If the client was already closed due to error, returns that error
-func (dc *DCPClient) Close() error {
+// Close is used externally to stop the DCP client.
+func (dc *GoCBDCPClient) Close() {
 	dc.close()
-	return dc.getCloseError()
```

Comment:
Do we not care about reporting any potential close error, or is that being done elsewhere?